app/views/carts/show.html.erb

```diff
@@ -21,4 +21,6 @@
 <h3>Total: <%= @basket.total_cost %></h3>
 <% if current_user %>
   <%= button_to "Checkout", user_orders_path(current_user) %>
+<% else %>
+  <%= button_to "Checkout", login_path %>
```

Comment:
@anon0mys @jrambold @jamesrnelson Is this the best way to implement this feature?  Or should the redirect be handled by a controller instead?

Reply:
If we change the name of the button I like the overall implementation. Maybe "Register to Checkout' or something?

Reply:
I like that idea.  I'll change it and re-push.